components/create_comment/create_comment.jsx

```diff
@@ -154,7 +159,7 @@ export default class CreateComment extends React.PureComponent {
             this.setState({draft: {...newProps.draft, uploadsInProgress: []}});
         }
 
-        if (!Utils.areObjectsEqual(this.props.draft, newProps.draft)) {
+        if (this.props.messageInHistory !== newProps.messageInHistory) {
```

Comment:
Why do we need to update the draft when the message history changes? I don't understand what this would be useful for since we're already setting it above when a new thread is opened from the RHS

Reply:
The messageInHistory is something like "messageInHistory pointer" this means, that I pressed CTRL+UP, so, I have to change the input field. The previous approach is to use the props.draft, which changes more frequenly and can generate some kind of interferences and race conditions with the localForage (throught redux-persist) and multi-tab storage syncronization.

Reply:
Oh, I see. I thought that only changed when you sent a message, not when you used that feature.